projects/quantlib/build.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -eu
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+mkdir build
+cd build
+export LDFLAGS="-Wl,-rpath,'\$ORIGIN/lib'"
+cmake .. -GNinja -DBOOST_ROOT=/usr -DCMAKE_BUILD_TYPE=Release -DQL_COMPILE_WARNING_AS_ERROR=ON -L
```

Comment:
You'll need to enable `QL_BUILD_FUZZ_TEST_SUITE` now that it's off by default, I guess?

Reply:
Oh yeah, that'd be why it's not working cheers. That's an easy enough fix.